tests/phpunit/tests/rest-api/rest-schema-setup.php

```diff
@@ -186,6 +186,8 @@ public function test_expected_routes_in_schema() {
 			'/wp-site-health/v1/tests/authorization-header',
 			'/wp-site-health/v1/tests/page-cache',
 			'/wp-site-health/v1/directory-sizes',
+			'/wp/v2/wp_pattern_category',
+			'/wp/v2/wp_pattern_category/(?P<id>[\d]+)',
```

Comment:
I wasn't able to run this test suite locally for some reason, but the earlier [CI failures](https://github.com/WordPress/wordpress-develop/actions/runs/6216587899/job/16870613378?pr=5234) pointed to these lines being missing so I added them in.

Reply:
Hmm, still failing. Does anything need to be updated on the REST API side to support this taxonomy?

Reply:
Will take a look

Reply:
This is all that was added in the plugin to make it work https://github.com/WordPress/gutenberg/pull/53163/files

Reply:
If I run this branch locally going to /wp-admin/edit-tags.php?taxonomy=wp_pattern_category shows the taxonomy and allows you to edit it, but that is not via REST I guess

Reply:
I ran all the unit tests locally and this regenerated [the fixtures file](https://github.com/WordPress/wordpress-develop/pull/5234/files#diff-1266de8ae81595637e30359637aa0cdd1a9f4d2644a8ec539a26f6dafd244fb6R4443) - so hopefully pushing this fixes it.